web-app/src/view/veilarbportefolje/veilarbportefolje.tsx

```diff
@@ -38,27 +38,28 @@ export function Veilarbportefolje() {
             />
             <AdminKnapp
                 tittel="Hovedindeksering"
-                beskrivelse="Oppdaterer alle brukere i opensearch. Tar typ 10min i produksjon."
+                beskrivelse="Oppdaterer alle brukere i opensearch. Tar ca 20 min i produksjon."
                 request={hovedindeksering}
             />
             <AdminKnapp
                 tittel="Hovedindeksering: Nytt alias"
                 beskrivelse="Skriver alle brukere til en ny indeks.
                 Alle oppdateringer til gammel indeks vil stanse mens jobben kjører.
                 Dette vil også skape noen warnings i loggene.
+                Kun nødvendig dersom det er gjort endringer på opensearch_settings.json.
```

Comment:
Ville lagt til en beskjed her om å tenke seg godt om. Dvs. denne operasjonen blokkerer oppdatering av brukere helt til den har fullført, og man må derfor ta med i betraktning kostnaden (forsinkelse) dette innebærer for veiledere.